CHANGELOG.md

```diff
@@ -32,6 +32,7 @@
 * Export profile picture for library card printing. Refs UIU-3160.
 * Bump up users okapi interface to 16.2. Refs UIU-3170.
 * Fix wrong format of user expiration date in User Edit. Refs UIU-3169.
+* Fix user input error in Preferred email communications field in User Edit. Refs UIU-3173.
```

Comment:
Please rephrase this to depict the solution - something to reflect filter options on typing